lib/std/time/timezone.zig

```diff
@@ -0,0 +1,683 @@
+const std = @import("std");
+
+pub const Timezone = struct {
+    offset: i16, // In minutes
+    name: []const u8,
+
+    // Auto register timezone
+    pub fn create(name: []const u8, offset: i16) Timezone {
+        const self = Timezone{.offset=offset, .name=name};
+        return self;
+    }
+};
+
+const create = Timezone.create;
+
+// Timezones
+pub const Africa = struct {
+    pub const Abidjan = create("Africa/Abidjan", 0);
```

Comment:
This part of the module is entirely incorrect. Timezones regularly change and we shouldn't embed them into the zig standard library.
Instead we need an interface to read+use tzfiles.